### DISCUSSION TITLE
Welcome to Wing Discussions!

### DISCUSSION THREAD

**Discussion by eladb** [winglang/wing#491] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by doctorfree** [comment]
Hi, I'm Ron Record, an open source developer working on custom Docker image builds for Kasm. I've just started working on a Docker build for Wing.

**Reply by eladb** [reply]
Awesome!

**Comment by staycoolcall911** [comment]
Nice @doctorfree! Looking forward to see what you'll come up with. If it's #767, then you can comment on the issue and I'll assign it to you. In any case we'll be happy to assist, so just let us know if you have any questions either here or on our [Slack `#dev` channel](https://t.winglang.io/slack).

**Comment by ghost** [comment]


**Reply by revitalbarletz** [reply]
Hi @lucasnuic ! Great to have you here! How about joining our slack channel https://t.winglang.io/slack and be part of our community?
